UPGRADING-2.0.md

```diff
@@ -199,7 +199,7 @@ Upgrading From 1.x To 2.0
       format_listener:
           rules:
               - { path: ^/, fallback_format: json, attributes: { _controller: FOS\RestBundle\Controller\ExceptionController } }
-              - { path: ^/, fallback_format: html } }
```

Comment:
you mean the space at the end of the line right? Just removed it in the follow-up commit.